incubator/hnc/test/e2e/issues_test.go

```diff
@@ -225,4 +225,15 @@ var _ = Describe("Issues", func() {
 		runShouldNotContain("CannotPropagateObject", 1, "kubectl hns describe", nsParent)
 		runShouldNotContain("CannotUpdateObject", 1, "kubectl hns describe", nsChild)
 	})
+
+	It("Should propogate admin rolebindings - issue 772", func(){
+		// set up
+		mustRun("kubectl create ns", nsParent)
+		mustRun("kubectl create ns", nsChild)
+		mustRun("kubectl hns set", nsChild, "--parent", nsParent)
+		// Creating admin rolebinding object and waiting 2s
+		mustRun("kubectl create rolebinding --clusterrole=admin --serviceaccount=default:default -n", nsParent, "foo")
+		// Object should exist in the child, and there should be no conditions
+		mustRun("kubectl get rolebinding foo -n", nsChild, "-oyaml")
```

Comment:
Just double checking, if you remove line 235, can this line pass?

What I want to see is that if the rolebinding resource is not there and the `k get rolebinding` will pass or fail the `mustRun()`.

Reply:
Yes. When I had a typo in line 235 before and this test kept failing until I corrected it.

Reply:
Ah what was the typo? Just wanted to see if the test failed on line 235 because of the typo or line 237 because the resource is not there.

Reply:
I wrote it as `--clusterrole=cluster-admin`

Reply:
Ah it means it will return an error when `mustRun(k get)` if the resource doesn't exist. Thanks for confirming!